sources-grammars.ts

```diff
@@ -648,6 +648,12 @@ export const sourcesCommunity: GrammarSource[] = [
     displayName: 'Mojo',
     source: 'https://github.com/modularml/mojo-syntax/blob/main/syntaxes/mojo.syntax.json',
   },
+  {
+    name: 'move',
+    displayName: 'Move',
+    aliases: ['mv'],
```

Comment:
I don't see `mv` has been used https://github.com/github-linguist/linguist/pull/5955/files#diff-9c337ec09ccaaa1374713aad9ba26cdc1308586a27cb309a883a0b19bde663d8R4070, do you have a reference?

Reply:
`.mv` is the bytecode format of the `move` source code (i.e., https://github.com/diem/move/blob/main/language/move-binary-format/src/unit_tests/empty_script.mv)

Reply:
If you would prefer I not include aliases for it I can remove it?

Reply:
So actually `.mv` is different from `.move`, right? In that sense I think we could not include the alias

Reply:
You got it boss 👍